src/api/namespaces/helpers/handlers/collections.ts

```diff
@@ -1,24 +1,37 @@
 import {filterQueryArgs, RequestValues} from '../../utils';
 import {NeftyMarketContext} from '../index';
 import QueryBuilder from '../../../builder';
+import { ApiError } from '../../../error';
 
 export async function getCollectionsAction(params: RequestValues, ctx: NeftyMarketContext): Promise<any> {
     const args = filterQueryArgs(params, {
-        list: {type: 'string', values: ['whitelist', 'blacklist', 'verified', 'nsfw', 'scam']},
+        lists: {type: 'string' },
         sort: {type: 'string', values: ['collection_name', 'list'], default: 'collection_name'},
         order: {type: 'string', values: ['asc', 'desc'], default: 'asc'},
     });
 
-
+    // @TODO: index (collection_name, list)
     const query = new QueryBuilder(`
-                SELECT collection_name, contract, list
+                SELECT collection_name, list
                 FROM helpers_collection_list as v
             `);
+
     query.equal('v.assets_contract', ctx.coreArgs.atomicassets_account);
-    if (args.list) {
-        query.equal('v.list', args.list);
+    if (args.lists) {
+        const lists = args.lists.split(',');
+        for(const list of lists){
+            if(
+                list !== 'whitelist' && list !== 'blacklist' &&
+                list !== 'verified' && list !== 'nsfw' &&
+                list !== 'scam'
+            ){
+                throw new ApiError(`Invalid value ${list} for parameter lists`, 400);
+            }
+        }
+        query.equalMany('v.list', lists);
     }
     query.append(`
+                GROUP BY collection_name, list
```

Comment:
grouping by list will still return the collection more than once, or is not the goal to return only one time per collection?